docker/Dockerfile.multiarch

```diff
@@ -0,0 +1,15 @@
+FROM --platform=$BUILDPLATFORM golang:1.16 AS build
```

Comment:
we should delete other docker files as they would not work anymore anyway

Reply:
removed files